default.nix

```diff
@@ -55,6 +55,10 @@ let
     regeneratePackages = commonLib.pkgsDefault.callPackage ./nix-tools-regenerate.nix {
       nix-tools = package;
     };
+    # default and release templates that abstract
+    # over the details for CI.
+    default-nix = import ./nix-tools-default.nix commonLib;
+    release-nix = import ./nix-tools-release.nix commonLib;
```

Comment:
Are these usable as part of a larger `default.nix` or `release.nix`? For example, in the Plutus repo we also have a bunch of latex that's buildable via our top-level `default.nix` and included in `release.nix`.

Also, writing a little about how to use these in a README or similar would be good.

Reply:
It should be perfectly fine to use the `default-nix` template and just merge it with custom package definitions. The `release-nix` will still rely on the `default.nix`.

Reply:
@michaelpj does the example (See the PR message atop) suite your needs?

Reply:
Good example, do put them somewhere in the actual repository so they're available for future people!